wrangler-core/src/test/java/co/cask/directives/external/InvokeHttpTest.java

```diff
@@ -49,16 +48,16 @@
  */
 public class InvokeHttpTest {
   private static NettyHttpService httpService;
-  protected static String baseURL;
+  private static String baseURL;
 
   @Before
   public void startService() throws Exception {
     List<HttpHandler> handlers = new ArrayList<>();
     handlers.add(new ServiceHandler());
     httpService = NettyHttpService.builder("Services")
```

Comment:
Should have a direct dependency on the netty-http library.

Reply:
I think its already there. Just need to update the version...